package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "azure-devops-node-api",
     "description": "Node client for Azure DevOps and TFS REST APIs",
-    "version": "9.0.3",
+    "version": "9.1.0",
```

Comment:
Major version bump?

Reply:
I'm never sure whether we should go major or minor. In theory this shouldn't be breaking, right? Idk, we need to figure out a good plan here. I'm fine bumping to major for now, I'll do that

Reply:
Done